khal/controllers.py

```diff
@@ -21,6 +21,8 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
Maybe from a readability POV (although a very weak point IMO): Because only some modules have that future-import, you can't know whether a particular string literal produces a unicode or bytestring. So we should then do this import for *every* module, but that means we'd have to wrap 70% of string literals with `str(...)` (because "native strings" is what most stdlib APIs actually expect)